app/controllers/core_induction_programme/lessons_controller.rb

```diff
@@ -1,7 +1,36 @@
 # frozen_string_literal: true
 
 class CoreInductionProgramme::LessonsController < ApplicationController
+  include Pundit
+
+  after_action :verify_authorized, except: :show
+  before_action :authenticate_user!, except: :show
+
   def show
     @course_lesson = CourseLesson.find(params[:lesson_id])
   end
+
+  def edit
+    @lesson = CourseLesson.find(params[:lesson_id])
+    authorize @lesson
+
+    @lesson_preview = params[:lesson_preview] || @lesson.content
+    @course_lesson = Govspeak::Document.new(@lesson_preview, options: { allow_extra_quotes: true }).to_html
```

Comment:
We might want to make a helper, say `GovspeakHelper` and add a method there `content_to_html` that will do this lengthy thing for us - we duplicate it in a bunch of places and it's getting annoying.

Reply:
No problem, will add that in.

i'll update the govspeaktest controller while i've added it.

Reply:
Not sure exactly what's going on here. I find it a little surprising that `@lesson` is a `CourseLesson`, while `@course_lesson` is a... something else?

Reply:
Especially since `@course_lesson` is a `CourseLesson` in `show`

Reply:
I think this is just inconsistent naming on my part. @course_lesson and @lesson are the same thing. Will stick with one.